common/testing_stubs.go

```diff
@@ -0,0 +1,47 @@
+package common
```

Comment:
Actually maybe this could incorporated into our existing `testutil.go` ? I also wonder if it'd be good to rename that file `util_test.go` so we aren't compiling that code in by default (or rename this one to stub_test.go`).

Reply:
I'll move it into `testutil.go`, but it can't be ranamed to `util_test.go` - or else I could have been able just use `core.StubServerStream` from my new test.

Reply:
Ah, test files from other modules aren't incorporated? Bummer, alright.